src/main/kotlin/com/zepben/evolve/testing/TestNetworkBuilder.kt

```diff
@@ -21,7 +21,7 @@ import kotlin.reflect.full.primaryConstructor
 /**
  * A class for building simple test networks, often used for unit testing.
  */
-open class TestNetworkBuilder {
+internal open class TestNetworkBuilder {
```

Comment:
urgent: This must remain a public class as it is very widely used by migrators. It was deliberately made `open` to allow people that want to add CIM extension classes to extend this builder to help them build their own object, but that may have been alleviated by the improvements to `fromOther` etc, so happy for that to be removed if you insist.

Reply:
Removed `internal`.